editor/src/clj/editor/model_scene.clj

```diff
@@ -372,15 +372,23 @@
 (defn- make-scene [renderable-mesh-set model-scene-resource-node-id]
   (let [{:keys [aabb renderable-models]} renderable-mesh-set
         model-scenes (mapv #(make-model-scene % model-scene-resource-node-id)
-                           renderable-models)]
+                           renderable-models)
+        children-scenes (into [{:node-id model-scene-resource-node-id
+                                :aabb aabb
+                                :renderable {:render-fn render-outline
+                                             :tags #{:model :outline}
+                                             :batch-key nil
+                                             :select-batch-key :not-rendered
+                                             :passes [pass/outline]}}]
+                              model-scenes)]
     {:node-id model-scene-resource-node-id
      :aabb aabb
      :renderable {:render-fn render-outline
-                  :tags #{:model :outline}
+                  :tags #{:model}
                   :batch-key nil ; Batching is disabled in the editor for simplicity.
                   :select-batch-key :not-rendered ; The render-fn only does anything during the outline pass.
-                  :passes [pass/outline pass/opaque-selection]} ; Include in a selection pass to ensure it can be selected and manipulated.
-     :children model-scenes}))
```

Comment:
We use the same tags for both passes: `pass/outline` and `pass/opaque-selection`, but `:outline` is off by `:toggle-component-guides`, so we should use different tags for these passes

Reply:
You should probably remove the `:render-fn` and `:select-batch-key` entries from the `:renderable` here too, based on the comments? 🤔

Reply:
done!

Reply:
No, no! You didn't do what I asked! You broke it now.

Reply:
ah, yeah
sorry
done now

Reply:
Dammit, I got it wrong when I wrote my "clarifying explanation". 😞 The entries should be removed from the `opaque-selection` pass, not the `outline` pass, because `;; The render-fn only does anything during the outline pass.`

It's still broken. Sorry about that.

Reply:
It's not your fault that I don't use my own brain here. Pros and Cons of code reviews.

Reply:
I carefully re-read everything. Removed 2 lines and fixed one comment.
Tested it again to make sure it works.
I think now it's fine